common-rest/src/main/java/org/uniprot/api/rest/output/converter/JsonMessageConverter.java

```diff
@@ -78,7 +85,32 @@ protected void before(MessageConverterContext<T> context, OutputStream outputStr
     @Override
     protected void writeEntity(T entity, OutputStream outputStream) throws IOException {
```

Comment:
I think it might be interesting to see what jvisualvm, or java mission control says when writing a big result set? Usually I wouldn't suggest this, but since this method is going to be called a lot, any performance improvements here could be useful.

I wouldn't spend much time doing this, but just to see whether anything stands out and needs optimising a bit.